Yes, ovemap, yes, again

comma opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

Adds ability to pick landing area for shuttles.
Occupied areas and those too small for shuttle to fit in are not selectable.

comma committed:
Yes, ovemap, yes, again

Adds ability to pick landing area for shuttles.
Occupied areas and those too small for shuttle to fit in are not selectable.

PsiOmegaDelta commented on code/game/area/areas.dm:
```diff
@@ -338,3 +338,22 @@ var/list/mob/living/forced_ambiance_list = new
+
+/area/proc/get_dimensions()
+	var/list/res = list("x"=1,"y"=1)
+	var/list/min = list("x"=255,"y"=255)
```
Might be worthwhile to use world.maxx and maxy

comma replied:
oh yeah, thought about sticking INF there, forgot about those derp

comma committed:
Pushes up the limits

tdtt LIIMIIITS
Uses maximum map values instead of declaring that 255 is the sky
